data/network/src/commonMain/kotlin/social/androiddev/common/network/model/request/CreateAccessTokenBody.kt

```diff
@@ -7,14 +7,17 @@
  *
  * You should have received a copy of the GNU General Public License along with Dodo. If not, see <https://www.gnu.org/licenses/>.
  */
-package social.androiddev.common.persistence
+package social.androiddev.common.network.model.request
 
-import com.squareup.sqldelight.db.SqlDriver
-import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
 
-actual suspend fun provideDatabaseDriver(schema: SqlDriver.Schema): SqlDriver {
-    // TODO: replace with AndroidSqliteDriver and inject context
-    return JdbcSqliteDriver(url = JdbcSqliteDriver.IN_MEMORY).also { driver ->
-        schema.create(driver = driver)
-    }
-}
+@Serializable
+internal data class CreateAccessTokenBody(
+    val scope: String,
+    val code: String,
+    @SerialName("client_id") val clientId: String,
```

Comment:
nit/stylistic https://publicobject.com/2016/01/20/strict-naming-conventions-are-a-liability/

Reply:
Hm - I might be missing the point here, but the blog/post mentions using the solution I have here (with annotations) as one of the _better_ alternatives for parsing `snake_case` json properties.

Reply:
lately I've been going with solution 3 of just leaving it but this is fine too